bin/main.dart

```diff
@@ -422,13 +423,16 @@ Future<void> _translateMicrosoft(List<String> stringInOutList,
       sourceLang +
       '&to=' +
       targetLang;
-  final headers = {
+  var headers = {
```

Comment:
```suggestion
  final headers = {
```

Reply:
Here headers is not final, because of the following line
headers.addAll({'Ocp-Apim-Subscription-Region': msRegion});

Reply:
Hm, does it complain when you put final?

I have in one project this:
final arguments = <String, dynamic>{
      'identifier': type.identifier,
    };
    arguments.addAll(predicate.map);

No complains =)

Reply:
You are right. We do not change headers.